sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/timeExpressions.scala

```diff
@@ -297,6 +312,36 @@ object HourExpressionBuilder extends ExpressionBuilder {
   }
 }
 
+case class SecondsOfTimeWithFraction(child: Expression)
+    extends RuntimeReplaceable
+    with ExpectsInputTypes {
```

Comment:
Please, fix indentation, see https://github.com/databricks/scala-style-guide?tab=readme-ov-file#spacing-and-indentation
```suggestion
  extends RuntimeReplaceable
  with ExpectsInputTypes {
```

Reply:
Done